jobs/prometheus2/templates/bin/prometheus_ctl

```diff
@@ -79,7 +79,9 @@ case $1 in
       --web.console.templates="<%= templates %>" \
       <% end %> \
       <% if_p('prometheus.web.enable_lifecycle') do |enable_lifecycle| %> \
-      --web.enable-lifecycle="<%= enable_lifecycle %>" \
+      <% if p('enable_lifecycle') %> \
```

Comment:
This fails if the property is not set. You need to add a `default: false` at the [property definition](https://github.com/bosh-prometheus/prometheus-boshrelease/blob/master/jobs/prometheus2/spec#L80)